src/AppBundle/Repository/EventRepository.php

```diff
@@ -236,14 +263,51 @@ private function getFileUsageBuilder(string $dbName, DateTime $start, DateTime $
         }
 
         $rqb->andWhere('img_timestamp BETWEEN :start AND :end')
-            ->andWhere('img_user_text IN (:usernames)')
             ->setParameter('start', $start)
             ->setParameter('end', $end)
             ->setParameter('usernames', $usernames, Connection::PARAM_STR_ARRAY);
 
+        $this->applyUsersOrCategories($rqb, $dbName, $usernames, $categories);
+
         return $rqb;
     }
 
+    /**
+     * @param QueryBuilder $rqb
+     * @param string $dbName
+     * @param string[] $usernames
+     * @param string[] $categories
+     */
+    private function applyUsersOrCategories(
+        QueryBuilder $rqb,
+        string $dbName,
+        array $usernames,
+        array $categories
+    ): void {
+        if ($usernames && $categories) {
+            // throw new InvalidParameterException('Cannot process events with both users and categories');
```

Comment:
Code is commented out; but anyway we do allow participants + categories, so I don't think we need this check.

Reply:
Removed.